Taarafo.Core/Services/Foundations/Posts/PostService.Validations.cs

```diff
@@ -93,6 +109,21 @@ private bool IsDateNotRecent(DateTimeOffset date)
 
             return timeDifference.Duration() > oneMinute;
         }
+        private static void ValidateAginstStoragePostOnModify(Post inputPost, Post storagePost)
+        {
+            switch (inputPost)
+            {
+                case { } when inputPost.CreatedDate != storagePost.CreatedDate:
+                    throw new InvalidPostException(
+                        parameterName: nameof(inputPost.CreatedDate),
+                        parameterValue: inputPost.CreatedDate);
+
+                case { } when inputPost.UpdatedDate == storagePost.UpdatedDate:
+                    throw new InvalidPostException(
+                        parameterName: nameof(inputPost.UpdatedDate),
+                        parameterValue: inputPost.UpdatedDate);
```

Comment:
This is not The Standard 2.0 for validations

Reply:
How should I upgrade this method?

Reply:
For modify scenarios you can do this:
https://github.com/hassanhabib/OtripleS/blob/master/OtripleS.Web.Api/Services/Foundations/Fees/FeeService.Validations.cs#L46